### ISSUE TITLE
Game Chunks Load Very Slow

### ISSUE THREAD

**Issue by PolyframeX** [AstralOrdana/Spelunkery#54]
**Describe the bug:**
The mod causes major lag, onto where chunks can take up to 10 seconds to load a single chunk.
![image](https://user-images.githubusercontent.com/80133652/225097929-3679b20b-14d8-4f07-be01-e2cfe890635f.png)

**Versions: (BEFORE SUBMITTING A BUG REPORT, make sure you have the most up-to-date versions of Spelunkery, Moonlight Lib and Supplementaries)**
Minecraft version: 1.19.2
Spelunkery version: 0.1.13
Moonlight Lib version: 2.2.19
Fabric API / QSL / Forge version: 0.75.1

**Other mods:**
Sodium 0.4.4

**Logs:**
https://pastebin.com/9sWAMaCN

**To Reproduce:**
Steps to reproduce the behavior. Format it as a numbered list of steps.

1. Download Fabric, Preferably 1.19.2
2. Download Spelunkery w/ Dependencies
3. Load A Normal Minecraft World

**Expected behavior:**
Game Should Load Minecraft Chunks Fast And Normal

**Comment by kyragit** [comment]
I know this is a known issue, but I experienced this too and I found the root cause. It looks like `FastNoiseLite.GetNoise()`(lol) is mainly causing the performance drop, along with minecraft's `LevelReader.getBiome()`. 
![image](https://user-images.githubusercontent.com/87741177/225132557-33f07c6f-3827-4cbe-9e59-83cd759c0785.png)
Here's a spark profile of the offenders. Hope this helps, I like the mod!

**Comment by grundyboy34** [comment]
Yeah I was having this issue on my server too. Although I couldn't get a spark profiler to give much info, I made a replica server and used chunky to keep generating the same seed, so I had a constant to test against.
Just the plain fabric 1.19.2 install would generate 35 chunks per second. With all my mods, that dropped to 9 - 12 chunks per second. Once I disabled spelunkery's worldgen in the config, it jumped up to ~24 chunks per second.

**Comment by SaltSouls** [comment]
I've also been getting generation lag, but just looking at the console for me, it would seem it might be because the game is trying to place features in faraway unloaded chunks and is erroring.
![ksnip_20230319-185129](https://user-images.githubusercontent.com/36361103/226228904-f05d64e3-5e7f-4652-aabf-09238e22be88.png)

**Comment by AstralOrdana** [comment]
those error messages are unrelated to the chunk lag. The lag has been fixed in the dev version and will be available in the next released version

**Comment by AstralOrdana** [comment]
fixed for 0.2.0
